src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -62,11 +62,19 @@
 (defn create-form-label [text]
   [:div {:style {:marginBottom "0.16667em" :fontSize "88%"}} text])
 
+;; jQuery3+React warns about nil :value elements. Replace with empty string.
+(defn- replace-nil-value-key [props]
+  (reduce-kv (fn [m k v]
+               (if (and (= k :value) (nil? v))
+                 (assoc m k "")
+                 (assoc m k v)))
+             {} props))
```

Comment:
Correct, we're only using this approach in `create-text-field` and `create-search-field` so it's very limited. We *may* need to use this in other fields, but I did not find any cases like that.

The other strange thing about this that I don't understand, and one reason I tried to keep this change very isolated, is that it does not happen with jquery2+react.